io/zenoh-transport/tests/unicast_openclose.rs

```diff
@@ -470,6 +470,18 @@ fn openclose_ws_only() {
     task::block_on(openclose_transport(&endpoint));
 }
 
+#[cfg(feature = "transport_shm")]
+#[test]
```

Comment:
I believe there is no need to run this test every time on the CI (it will make increase its duration).
Mark the test as follows:
```
#[test]
#[ignore]
```
It can be then run with `cargo test -- --ignored`.
Please apply the changes to `transport_ws` and `transport_unixsock-stream` as well.